software/messaging/src/test/java/brooklyn/entity/messaging/rabbit/RabbitEc2LiveTest.java

```diff
@@ -81,7 +82,16 @@ private Channel getAmqpChannel(RabbitBroker rabbit) throws Exception {
         Channel channel = conn.createChannel();
         return channel;
     }
-    
+
+    @Override
+    public void test_CentOS_5_6() throws SkipException {
+        // Not supported. The EPEL repository described here at [1] does not contain erlang, and the
```

Comment:
Agree with @sjcorbett `throw new SkipException()` means we don't pretend it worked in the test results.